native/packages/react-native-bpk-component-button/src/BpkButton-test.common.js

```diff
@@ -138,15 +164,9 @@ const commonTests = () => {
   });
 
   describe('BpkButtonThemed', () => {
-    const themeAttributes = {
-      gradientStartColor: '#CE93D8',
-      gradientEndColor: '#AB47BC',
-      textColor: 'rgba(255, 255, 255, 0.8)',
-    };
-
     it('should render correctly', () => {
       const tree = renderer.create(
-        <BpkThemeProvider theme={themeAttributes}>
+        <BpkThemeProvider theme={{}}>
```

Comment:
Why are we not testing this anymore?

Reply:
When I looked back at this, it wasn't actually testing it very accurately as the theme names changed (e.g. `textColor` became `buttonPrimaryTextColor`. So I figured it was less misleading to have it take an empty object.

Reply:
We need a test case to show that when we wrap a themeable component in a themeprovider with a theme object that it works though, right?

Reply:
Now addressed